.github/workflows/ci.yml

```diff
@@ -264,7 +264,7 @@ jobs:
       - name: Build bevy
         # this uses the same command as when running the example to ensure build is reused
         run: |
-          TRACE_CHROME=trace-alien_cake_addict.json CI_TESTING_CONFIG=.github/example-run/alien_cake_addict.ron cargo build --example alien_cake_addict --features "bevy_ci_testing,trace,trace_chrome"
+          TRACE_CHROME=trace-alien_cake_addict.json CI_TESTING_CONFIG=.github/example-run/testbed_2d.ron cargo build --example alien_cake_addict --features "bevy_ci_testing,trace,trace_chrome"
```

Comment:
```suggestion
          TRACE_CHROME=trace-alien_cake_addict.json CI_TESTING_CONFIG=.github/example-run/testbed_2d.ron cargo build --example testbed_2d --features "bevy_ci_testing,trace,trace_chrome"
```

Reply:
These pre-build steps are also still referencing alien cake addict with `TRACE_CHROME=trace-alien_cake_addict.json`.

Reply:
I'll remove the prebuilt steps, they're "fun" but not very useful